### PR TITLE
Updated readme info

### PR DESCRIPTION
**Pull request by TonysCousin**
# PR Details
## Description
Updated Readme to show new architecture diagram.
Updated the roadmap page description and it now links to the current roadmap diagram in Confluence.
Added several names to the attribution list.

## Motivation and Context

Improves relevance and provides routine updates for more accurate public representation.

## How Has This Been Tested?

Viewed the pages on the feature branch through the Github web site.

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](Contributing.md) 
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by TonysCousin** [comment]
@snallamothu feel free to approve and merge whenever you are ready. If I get updates from Ben beforehand, I"ll just add them as a new commit, and they'll be automatically included in what you review.

**Comment by msmcconnell** [comment]
@TonysCousin You actually want to commit to develop (not master) for this to show up. That's our base branch.

**Comment by TonysCousin** [comment]
@msmcconnell No, I want to merge to master.  Then master will get merged into develop. In that way, the latest documentation will always be visible on master, where people may be pulling a tagged version.  If we do it on develop then it may not percolate into master for many weeks.

File: docs/Roadmap.md [hunk 1]
```diff
@@ -1,16 +1,11 @@
 # CARMA Roadmap
 
-*[Updated 28 Jul 2019]*
+*[Updated 10 Jan 2020]*
 
 This page presents the current vision for future release activity of the CARMA platform.  It is focused on version 3 (CARMA3) since that is a major upgrade.
-Beyond that the future is less clear, although we will definitely be pushing the system toward SAE level 4 autonomy while continuing to extend the sophistication of the V2x communication capabilities and resultant cooperative behavior.
+Beyond that we will definitely be pushing the system toward SAE level 4 autonomy while continuing to extend the sophistication of the V2x communication capabilities and resultant cooperative behavior.
```
Comment (snallamothu) [inline docs/Roadmap.md]: V2x needs to be V2X

**Pull request merged**